src/app/screens/LNURLPay/index.tsx

```diff
@@ -497,6 +498,7 @@ function LNURLPay() {
                           <div className="mt-4">
                             <TextField
                               id="name"
+                              type="text"
```

Comment:
I think type text is default

Reply:
we have type="text" at few of the text field and few of them doesn't contain them.  it's better to be consistent!

shall we remove it from everywhere or add it everywhere? isn't this better and readable to have type explicitly defined